plugins/mcreator-localization/lang/texts.properties

```
@@ -1032,6 +1032,8 @@ blockly.block.item_can_harvest=Can %3 %1 harvest %4 %2
 blockly.block.item_can_smelt=Can %1 be smelted
 blockly.block.item_check_rarity=Is %1 of rarity %2
 blockly.block.item_cooldown_for=Cooldown %3 for %1 ticks for %2
+blockly.block.item_cooldown_get=Remaining cooldown percent of %2 for %1
```

I think the text should be something like this to follow what other getters use 
```suggestion
blockly.block.item_cooldown_get=Get remaining cooldown percentage of %2 for %1
```